src/vue/components/popup.vue

```diff
@@ -46,7 +46,11 @@
             <close-icon></close-icon>
           </div>
       </div>
-
+      <div class="alpheios-popup__toolbar-buttons alpheios-popup__toolbar-closeonly" v-show="! showToolbar">
+        <div @click="ui.closePopup" class="alpheios-popup__close-btn">
+          <close-icon></close-icon>
+        </div>
+      </div>
```

Comment:
I'm wondering if the code be simpler if we hide just the action buttons inside the toolbar by wrapping them in something like a `template` or a `div` with a `v-show` conditional?